Userland/Applications/PixelPaint/FilterParams.h

```diff
@@ -107,8 +107,8 @@ struct FilterParameters<Gfx::SpatialGaussianBlurFilter<N>> {
 
         for (auto x = -offset; x <= offset; x++) {
             for (auto y = -offset; y <= offset; y++) {
-                auto r = sqrtf(x * x + y * y);
-                kernel.elements()[x + offset][y + offset] = (expf(-(r * r) / s)) / (float { M_PI } * s);
+                auto r = AK::hypot((float)x, (float)y);
```

Comment:
This seems like a precision losing call.

Reply:
x and y are ints and it used floats earlier too, so no loss in comparison to the past one,
in praxis the will be loaded directly onto the fpu stack, which can represent 64 bit numbers without loss